src/PAModel/CanvasDocument.cs

```diff
@@ -422,8 +426,85 @@ internal HashSet<string> GetImportedComponents()
             return set;
         }
 
+
+
+        private void StabilizeAssetFilePaths()
+        {
+            _entropy.LocalResourceFileNames.Clear();
+
+            // Update AssetFile paths
+            foreach (var resource in _resourcesJson.Resources)
+            {
+                if (resource.ResourceKind != "LocalFile")
```

Comment:
nit: can we put this string literal as a constant? als omakes it easier to comment it.

Reply:
yup